docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -88,7 +89,8 @@ Add a group using an object with `label` and `items` properties.
 The `label` will be used as the heading for the group.
 Add links or subgroups to the `items` array.
 
-```js
+```js /(label:) 'G/ /(label:) 'St/ "items:"
```

Comment:
I only wanted to highlight the `label` property of the groups and not the ones of the links too.

Reply:
Haha, nice to see someone finally using the regexp matching!

You could shorten the expression a bit though:
```suggestion
```js /^\s*(label:|items:)/
```

Reply:
I was a bit afraid of using `\s` with Prettier which could technically break this one, but seeing how clean your suggestion is, I think it's worth it and no matter which syntax we use, we would still need to make sure it's ok during the review of a new translation so we might as well use the cleanest one.

Thanks for the great suggestion 🤩

Reply:
Famous last words: I'm pretty confident that Prettier is not going to waltz in and make that expression not match the intended strings anymore. :) As long as there is nothing but optional whitespace (spaces/tabs) in front of the keys, they should match.

And if we really do run into issues with Prettier, I'm looking forward to learning why and how to prevent them. :)

Reply:
I just tested it and the labels would need to be really long. Even `{ label: 'These are really long labels', link: '/guides/components/' },` still remains on one line. So let's hope for the best! :)